### ISSUE TITLE
Github link to tf.nest.map_structure is broken in the docs

### ISSUE THREAD

**Issue by chmendoza** [tensorflow/tensorflow#62922]
### Issue type

Documentation Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

2.8

### Custom code

No

### Current behavior?

The "View source on Github" button in https://www.tensorflow.org/api_docs/python/tf/nest/map_structure takes me to a file that doesn't exist. Also, the doc says "Refer to [tf.nest](https://www.tensorflow.org/api_docs/python/tf/nest) for the definition of a structure.", but if I go to that link, there is no such definition.

### Standalone code to reproduce the issue

```shell
n/a
```

### Relevant log output

```shell
n/a
```

**Comment by sushreebarsa** [comment]
@chmendoza You're right, there have been reports already of some TensorFlow API GitHub source code links not working correctly. Tensorflow has been released at 2.15.0.post1 ([see pypi](https://pypi.org/project/tensorflow/)), so the docs reflect like that. Seems like the repo is missing the appropriate tag for which an internal fix was raised. Thank you!

**Comment by chmendoza** [comment]
Apologies if creating noise, but here is another one with a broken link: https://www.tensorflow.org/api_docs/python/tf/nn/softmax

**Comment by sushreebarsa** [comment]
@chmendoza Yes, there are a few other apis are also there which are having the same issue as you mentioned. We are expecting it to be fixed in the next release. 
Thank you!
